### PR TITLE
Use new setup-oracle-action

### PR DESCRIPTION
**Pull request by andreasohlund**

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -20,9 +20,12 @@ jobs:
...
             RavenDB4,
             SqlServer,
+            Oracle,
+            Oracle-RabbitMQ,
             AzureStorage-RabbitMQ,
             AzureStorage-RavenDB4,
             AzureStorage-SqlServer,
+            AzureStorage-Oracle,
             RabbitMQ-RavenDB4,
             RabbitMQ-SqlServer
```
Comment (adamralph) [inline .github/workflows/ci.yml:30]: When we created this workflow, we tried to keep these in alphabetical order, within singular/multiple groups, and the multiple groups use the `transport-peristence` convention:

```suggestion
            Oracle,
            RabbitMQ,
            RavenDB4,
            SqlServer,
            AzureStorage-Oracle,
            AzureStorage-RabbitMQ,
            AzureStorage-RavenDB4,
            AzureStorage-SqlServer,
            RabbitMQ-Oracle,
            RabbitMQ-RavenDB4,
            RabbitMQ-SqlServer
```

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -88,14 +82,13 @@ jobs:
         with:
           connection-string-name: AzureStorageConnectionString
           tag: TimeoutMigrationTool
-      - name: Setup Oracle
+      - name: Install Oracle
```
Comment (adamralph) [inline .github/workflows/ci.yml:85]: Why change this? AFAIK we use the term "Setup" everywhere else.

File: .github/workflows/ci.yml [hunk 3]
```diff
@@ -88,14 +82,13 @@ jobs:
...
         if: contains(matrix.engines, 'Oracle')
-        shell: pwsh
-        run: |
-          echo "Adding environment variables for Oracle connection strings"
-          echo "OracleConnectionString=User Id=particular;Password=Welcome1;Data Source=127.0.0.1:1521/nservicebus;Enlist=dynamic" | Out-File -FilePath $Env:GITHUB_ENV -Encoding utf-8 -Append
+        uses: Particular/setup-oracle-action@a10e77929e10c7309a674e9dcf1597673027229a
```
Comment (adamralph) [inline .github/workflows/ci.yml:87]: AFAIK we use a version tag everywhere else.

File: .github/workflows/ci.yml [hunk 4]
```diff
@@ -17,12 +17,15 @@ jobs:
...
             AzureStorage,
+            Oracle,
             RabbitMQ,
             RavenDB4,
             SqlServer,
+            AzureStorage-Oracle,
             AzureStorage-RabbitMQ,
             AzureStorage-RavenDB4,
             AzureStorage-SqlServer,
+            RabbitMQ-Oracle,
```
Comment (adamralph) [inline .github/workflows/ci.yml:28]:
```suggestion
            Oracle-RabbitMQ,
```

**Review by danielmarbach — APPROVED** [review]

**Pull request merged**